.github/workflows/build-and-deploy.yml

```diff
@@ -35,11 +35,16 @@ jobs:
           yaml-lint -q -n $(find _posts -regex ".*.md\|.*html") &&\
           yaml-lint -q -n $(find pages -regex ".*.md\|.*html")
 
-      - name: Get GeoJSON for instructor map
-        run: |
-          curl --remote-name-all https://feeds.carpentries.org/all_instructors_by_airport.geojson &&\
-          cp all_instructors_by_airport.geojson files/.
+      - name: Setup EESSI to give us an R module
+        uses: eessi/github-action-eessi@v1
```

Comment:
All ye who use HPC, take note of this awesome action!

Reply:
This is giving us R with over 900 packages installed in just over 30 seconds!

Reply:
Super cool!